sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,309 @@
+# Guide for migrating to `azure-data-tables` from `azure-cosmosdb-table`
+
+This guide is intended to assist in the migration to `azure-data-tables` from `azure-cosmosdb-table`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+We assume that you are familiar with `azure-cosmosdb-table`. If not, please refer to the README for [`azure-data-tables`](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/tables/azure-data-tables#azure-data-tables-client-library-for-python) rather than this guide.
+
+## Table of contents
+
+* [Migration benefits](#migration-benefits)
+* [Important changes](#important-changes)
+    - [Package Names](#package-names-and-namespaces)
+    - [Client hierarchy and constructors](#client-hierarchy-and-constructors)
+    - [Authenticating Clients](#authenticating-clients)
+    - [Table Level Scenarios](#table-level-scenarios)
+    - [Entity Level Scenarios](#entity-level-scenarios)
+* [Additional Samples](#additional-samples)
+
+## Migration benefits
+
```

Comment:
The information provided in this section isn't really describing benefits to migration. I would describe this information as more historical or informational because it provides the rationale for the change. What it *does not* provide is a solid case for why a developer should take the time to migrate existing code to the new package. They presumably already have code that works, why change it? 

I suggest retitling this portion to something like "History" or "Why a new package?" and following up with a section titled "migration benefits" or "Why upgrade?" that discusses the bennies to a developer of moving to this new package.